### PR TITLE
Fixed docstrings for graph relabeling and sublattice mapping functionality missing from reference documentation.

### PR DESCRIPTION
**Pull request by qwriter**
Also, edited those docstrings.

This is the fix for issue https://github.com/dwavesystems/dwave-networkx/issues/234.

Duplicate of #237 because accidentally created PR from `main`.

### CODE REVIEW

**Comment by JoelPasvolsky** [comment]
An advantage of doing a rebase is that you don't get commits like [this one](https://github.com/dwavesystems/dwave-networkx/pull/238/commits/804006b1f043997eba485d310b3990e0f034c48d) scattered through the git history.

File: dwave_networkx/generators/chimera.py [hunk 1]
```diff
@@ -545,7 +557,7 @@ def __missing__(self, key):
 _chimera_coordinates_cache = __chimera_coordinates_cache_dict()
 
 def linear_to_chimera(r, m, n=None, t=None):
```
[the remark this answers is not in the document]
Comment (qwriter) [reply dwave_networkx/generators/chimera.py:555]: It looks like it is.

File: dwave_networkx/generators/chimera.py [hunk 2]
```diff
@@ -582,16 +594,16 @@ def linear_to_chimera(r, m, n=None, t=None):
 
 
 def chimera_to_linear(i, j, u, k, m, n=None, t=None):
```
[the remark this answers is not in the document]
Comment (qwriter) [reply dwave_networkx/generators/chimera.py:592]: It looks like it is.

File: dwave_networkx/generators/zephyr.py [hunk 3]
```diff
@@ -574,53 +592,58 @@ def zephyr_sublattice_mappings(source, target, offset_list=None):
...
     ----------
         source : NetworkX Graph
             The Chimera or Zephyr graph that nodes are input from.
         target : NetworkX Graph
             The Zephyr graph that nodes are output to.
         offset_list : iterable (tuple), optional (default None)
-            An iterable of offsets. This can be used to reconstruct a set of
-            mappings, as the offset used to generate a single mapping is stored
+            An iterable of offsets that can be used to reconstruct a set of
+            mappings since the offset used to generate a single mapping is stored
```
Comment (JoelPasvolsky) [inline dwave_networkx/generators/zephyr.py:624]:
```suggestion
            mappings. The offset used to generate a single mapping is stored
```

Here and in the other topologies: a 32-word sentence of this complexity is stretching it a bit.

**Review by JoelPasvolsky — COMMENTED** [review]
We currently have returned values for chimera functions but not pegasus & zephyr functions --- would be nice to have for all but I wouldn't add that to this PR.
Thanks!

**Review by JoelPasvolsky — APPROVED** [review]
LGTM, @qwriter, thank you!

**Pull request merged**